### ISSUE TITLE
markdown inside html comments can trigger warnings

### ISSUE THREAD

**Issue by ericseppanen** [rvben/rumdl#119]
I help maintain a newsletter in Markdown format. We often leave directions for contributors inside html `<!-- ... -->` comments. I am experimenting with rumdl but I'm having trouble because it is raising warnings for (and fmt tries to correct) code examples inside comments. We would prefer everything inside comments be ignored entirely.

For example, this file:
```markdown
# Hello world

<!--
  CFPs go here, use this format: * [project name - title of issue](URL to issue)
  * [ - ]()
-->
<!-- or if none - *No Calls for participation were submitted this week.* -->
```

Produces lots of warnings:

```plaintext
$ rumdl check contains_comments.md 
contains_comments.md:5:1: [MD005] Expected indentation of 0 spaces, found 2 [*]
contains_comments.md:5:1: [MD006] Consider starting bulleted lists at the beginning of the line (found 2 leading spaces) [*]
contains_comments.md:5:5: [MD039] Remove spaces inside link text [*]
contains_comments.md:5:5: [MD042] Empty link found: [ - ]()

Issues: Found 4 issues in 1 file (37ms)
Run `rumdl fmt` to automatically fix 3 of the 4 issues
```

Issue #20 discussed this, but it looks like comments are still not being ignored.

I tested rumdl 0.0.159 and 0.0.162.

**Comment by rvben** [comment]
Hi @ericseppanen,

Thanks for raising this issue. Content inside HTML comments should indeed be ignored by all linters.

This is now fixed and will land in the upcoming 0.0.163 release. Markdown syntax inside HTML comments (links, list items, images, etc.) will no longer trigger warnings.

Thanks again for making rumdl better. 👍
